test/Concurrency/Runtime/data_race_detection.swift

```diff
@@ -8,6 +8,7 @@
 // rdar://76038845
 // REQUIRES: concurrency_runtime
 // UNSUPPORTED: back_deployment_runtime
+// UNSUPPORTED: single_threaded_runtime
```

Comment:
There are two places below where specific line numbers are checked, and adding a line here breaks those. That's why the macOS tests failed.

Reply:
Oh, I didn't notice those lines. Thanks